app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_name, file_name, file_path, *_ = command.split()
```

Comment:
The `file_path` argument name is incorrect, you could use `file_name_out` or similar name to indicate that the variable's content is not only a path, but a filename as well